cmd/cmd.go

```diff
@@ -79,11 +78,5 @@ func initLogging() {
 }
 
 func logAppConfig() {
-	appCfgStr, err := yaml.Marshal(&appConfig)
-
-	if err != nil {
-		log.Debugf("Could not display application config: %+v", err)
-	} else {
-		log.Debugf("Application config:\n%+v", string(appCfgStr))
-	}
+	log.Debugf("Application config:\n%+v", appConfig.String())
```

Comment:
Something cool about Go:

By adding a receiver method onto a type (in this case, the `Application` struct type) with the signature _`String() string`_, you've actually implemented the `fmt.Stringer` interface! Stringers are cool, because the `fmt` functions treat them just as if they were plain old strings (by automatically calling `.String()` on your argument) when you use `%s`.

That means that this line of code can be simplified (a tad) to this:

```go
log.Debugf("Application config:\n%s", appConfig)
```